cmp/grid/columns/Column.js

```diff
@@ -658,6 +661,11 @@ export class Column {
         return null;
     }
 
+    // Normalize provided function or a boolean into a function form.
+    parseEditable(editable) {
+        return isFunction(editable) ? editable : () => !!editable;
```

Comment:
seems a bit weird to lose the ability for app to read the editable flag on the column later -- might be useful for troubleshooting or app code in the very common case that booleans are being used.